### DISCUSSION TITLE
第5节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#69] [笔记]
请在下方提交第 5 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135581429

**Comment by wocclyl** [comment]
第五课笔记：
https://m1ti1x9upsq.feishu.cn/wiki/PTzNw3rRuibC3skoynBc5WQ5nlf?from=from_copylink

**Comment by xiaomile** [comment]
[LMDeploy 的量化和部署](https://blog.csdn.net/weixin_42567071/article/details/135582707)

**Comment by wangxuejiao** [comment]
[LMDeploy 大模型量化部署实践](https://zhuanlan.zhihu.com/p/677779274)

**Comment by ShelterWFF** [comment]
class5: http://t.csdnimg.cn/f1zoU

**Comment by xingyuan3** [comment]
class5笔记：https://www.mubu.com/doc/3w9J3dSpB6d

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135585023

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/5.%20lmdeploy%20hands-on

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/05-61f07f4966474bbab3abe168b309e9f5

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135761093?spm=1001.2014.3001.5501
